The-Chonky-Panda/Atelier
# Rate review helpfulness

## Description

Added functionality to mark a single review as helpful or report a review.
Also merged in dev and wrap `<RatingsAndReviews />` with the click tracker.

## How to test


1. In each individual review, the `Yes` next to `Helpful?` should be clickable. Clicking on it should add 1 to the count in parentheses. Only the first time would add the count.
2. `Report` should also be clickable. Clicking on it would change it to `Reported`. It also can only be clicked on once. After 'reporting' a review, refresh the page, this review should disappear.

## Notes


## Issue tracking

https://trello.com/c/SIwPW7Rm
